docs/components/modeler/element-templates/element-templates.md

```diff
@@ -0,0 +1,7 @@
+---
+id: modeler-element-templates
```

Comment:
So long as this will be added before merging into main, I'm happy with this 👍

Reply:
Yes, the idea was to have smaller PRs to review the changes before pushing all the new content together to main. But if this complicates things, I can try a different approach.